run_unittests.py

```diff
@@ -1221,6 +1221,14 @@ def test_rpath_uses_ORIGIN(self):
             for path in rpath.split(':'):
                 self.assertTrue(path.startswith('$ORIGIN'), msg=(each, path))
 
+    def test_dash_d_dedup(self):
+        testdir = os.path.join(self.unit_test_dir, '10 d dedup')
+        self.init(testdir)
+        cmd = self.get_compdb()[0]['command']
+        self.assertTrue('-D FOO -D BAR' in cmd or \
```

Comment:
__[flake8]__
* [E502] the backslash is redundant between brackets

<!-- Comment by [SideCI](https://sideci.com) -->